static/js/containers/DashboardPage.js

```diff
@@ -686,6 +702,11 @@ class DashboardPage extends React.Component {
       />;
     }
 
+    let learnersInProgramCard;
+    if (R.pathOr(false, [program.id, 'data', 'learners'],programLearners)) {
+      learnersInProgramCard = <LearnersInProgramCard programLearners={programLearners[program.id].data}/>;
+    }
```

Comment:
what do you think about moving this out to a helper function? It could take `program` or `programID` and `programLearners` and return the `<LearnersInProgramCard />` with the right props passed in, so that we could just call that helper function in the JSX below, on like 759.